### PR TITLE
build distroless image using cekit in ci

### PR DESCRIPTION
**Pull request by matthyx**
As discussed in https://groups.google.com/g/quarkus-dev/c/cJaJJCowxHM/m/yxmF4eoUCQAJ
cc @cescoffier

### CODE REVIEW

File: quarkus-distroless-image.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- name: debian
+  version: stable-slim
+  from: debian:stable-slim
+
+# need to rename from image into "scratch" to disable /tmp cleanup
```
Comment (matthyx) [inline quarkus-distroless-image.yaml:5]: see here https://github.com/cekit/cekit/blob/master/cekit/templates/template.jinja#L208

**Comment by matthyx** [comment]
- libssl comes from gcr.io/distroless/base
- gcr.io/distroless/cc brings libc++
- I have to add libz from debian-slim

File: quarkus-distroless-image.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+- name: debian
+  version: stable-slim
```
Comment (cescoffier) [inline quarkus-distroless-image.yaml:2]: Why debian? Didn't you example use UBI?

**Review by cescoffier — CHANGES_REQUESTED** [review]

File: quarkus-distroless-image.yaml (hunk 2 above)
Comment (matthyx) [reply quarkus-distroless-image.yaml:2]: distroless images are built from debian, I think it's cleaner if the lib we're adding comes from the same distribution

**Comment by matthyx** [comment]
This cekit config replicates what's described here, but with a distroless/cc base image as we need libc++:
https://github.com/GoogleContainerTools/distroless/issues/217#issuecomment-404551174

File: quarkus-distroless-image.yaml (hunk 2 above)
Comment (cescoffier) [reply quarkus-distroless-image.yaml:2]: ah ok!

**Review by cescoffier — APPROVED** [review]

**Comment by matthyx** [comment]
@cescoffier is there a way to rerun tests... I think these failures are unrelated to my changes

**Comment by cescoffier** [comment]
Thanks a lot!

**Pull request merged**